libxw/src/libxw/legacy.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+from dataclasses import dataclass
+from typing import IO
+from typing import Optional
+from typing import Sequence
+
+
+# TODO: Configure actual logging...
```

Comment:
Unlike all this other Python infrastructural nonsense, logging is also a bit of a pain to get up and running.  I'll probably punt on doing this anytime soon :sweat_smile: